common/test/acceptance/tests/studio/test_studio_general.py

```diff
@@ -62,6 +71,145 @@ def test_dashboard_no_courses(self):
         self.dashboard_page.visit()
 
 
+class SignUpAndSignInTest(UniqueCourseTest):
+    """
+    Test studio sign-up and sign-in
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(SignUpAndSignInTest, self).setUp()
+        self.sign_up_page = SignupPage(self.browser)
+        self.login_page = LoginPage(self.browser)
+
+        self.course_outline_page = CourseOutlinePage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        self.course_outline_sign_in_redirect_page = CourseOutlineSignInRedirectPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        self.course_fixture = CourseFixture(
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run'],
+            self.course_info['display_name'],
+        )
+        self.user = None
+
+    def install_course_fixture(self):
+        """
+        Install a course fixture
+        """
+        self.course_fixture.install()
+        self.user = self.course_fixture.user
+
+    def test_sign_up_from_home(self):
+        """
+        Scenario: Sign up from the homepage
+        Given I visit the Studio homepage
+        When I click the link with the text "Sign Up"
+        And I fill in the registration form
+        And I press the Create My Account button on the registration form
+        Then I should see an email verification prompt
+        """
+        # HowitworksPage is the home page when user
+        # is not logged in. Visit it.
+        home_page = HowitworksPage(self.browser)
```

Comment:
@jzoldak I guess page DOM and content are the same for both howitworks page and page at URL "/". There is no readily available page for "/". Do you suggest to create a new one?